lib/resources/example_json/dstu2_examples_appointment.rb

```diff
@@ -412,5 +412,26 @@ module Resources
       "status": "proposed"
     }
 
+    DSTU2_APPOINTMENT_UPDATE ||= {
```

Comment:
For this example lets use a result from the sample search provided for Appointment search (https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment?date=2017&patient=4704007)

Reply:
Also it looks like a lot of the formatting is off here. Not sure how it would look on the site but I imagine it wouldn't look right.

Reply:
Addressed. Used actual result and fixed formatting.